docs/react-samples/src/App.tsx

```diff
@@ -32,8 +26,29 @@ function App() {
   return (
     <>
       <h1>Contact Center widgets in a react app</h1>
-      <StationLogin  onLogin={onLogin} onLogout={onLogout} />
-      <UserState />
+      <input
+        type="text"
+        placeholder="Enter your access token"
+        value={accessToken}
+        onChange={(e) => setAccessToken(e.target.value)}
+      />
+      <button
+        disabled={accessToken.trim() === ""}
+        onClick={() => {
+          store.init({webexConfig, access_token: accessToken}).then(() => {
+            setIsSdkReady(true);
+          });
+        }}
+      >Init Widgets</button>
```

Comment:
This is a sample app change. Not going to do it. This will make it obligated to do the same thing on Web Component Samples. Will become overhead and an overkill for a sample app